README.MD

```diff
@@ -1,11 +1,11 @@
 # justlog [![Build Status](https://github.com/gempir/justlog/workflows/ci/badge.svg)](https://github.com/gempir/justlog/actions?query=workflow%3Aci)
 
 ### What is this?
-Justlog is an twitch irc bot. It focuses on logging and providing an api for the logs.
+Justlog is a twitch irc bot. It focuses on logging and providing an api for the logs.
```

Comment:
May aswell capitalise T for Twitch as it's a proper noun. Same for IRC (it's an acronym)

Reply:
:nerd_face: